project-koku/koku
# remove update a source from api docs

Fixes [COST-2068](https://issues.redhat.com/browse/COST-2068)

Changes proposed in this PR:
* Removes update a source from Koku's public api docs 

Testing Instructions:
1. Access our api docs json file at docs/specs/openapi.json 
2.Copy the file and paste the contents into https://editor.swagger.io/
3. Check in the Sources section of the docs that add a source has been removed. 
Before 
![image](https://user-images.githubusercontent.com/54587293/175318576-0303d575-ae09-4fed-ab3c-3465c73546de.png)
After 
![image](https://user-images.githubusercontent.com/54587293/175318724-5e325d5a-4480-4911-9fbc-3d138f488c6a.png)




---
Additional Context:
Reach out if you need any help with testing!
